TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/ScoringAlgorithm.java

```diff
@@ -53,19 +59,37 @@ private void scoring() {
 
         setJointPos();
 
+
         while (pixelsInClaw) {
             robot.<Servo>get("clawServo").setPosition(robot.<Servo>get("clawServo").getPosition() - 0.125);
-        }
 
+        }
         resetPos();
 
-        telemetry.addData("Scoring Status", "Scored");
+    }
+
+    public void setViperSlidePos() {
+        telemetry.addData("Scoring Status", "Setting VS Pos");
+        // Setting ViperSlidePoses based on Targets.
+        while (Math.abs(robot.<DcMotor>get("leftViperSlideMotor").getCurrentPosition() - viperSlideTargetPos) > viperSlideTolerance) {
```

Comment:
Checking the left viper slide values to run both slides is a bad way to do this; if there's any difference in the speeds of the motors, then the difference is going to compound and you're going to end up with inaccurate parts at best, broken parts at worst.

Reply:
fixed...